src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Extension/Roslyn.SyntaxVisualizer.Extension.csproj

```diff
@@ -15,6 +15,7 @@
     <ExtensionInstallationFolder>Microsoft\SyntaxVisualizer</ExtensionInstallationFolder>
     <VisualStudioInsertionComponent>Roslyn.SyntaxVisualizer.Extension</VisualStudioInsertionComponent>
     <Ngen>false</Ngen>
+    <UseCodeBase>true</UseCodeBase>
```

Comment:
I'm a bit confused here by `CreateVsixContainer` being `false`. This project doesn't build VSIX?

Reply:
No, the Roslyn.SDK.2017 project embeds this dll inside of it.  It used to be a VSIX at some point.  I need to go refactor this to be more clear.

Reply:
So, then should `VisualStudioInsertionComponent`, `Ngen`, `ExtensionInstallationFolder`, etc. be set?

Reply:
Unsetting them causes the assemblies to be NGEN'd

Reply:
Setting `VisualStudioInsertionComponent` does. If this project doesn't produce VSIX then it shouldn't set `VisualStudioInsertionComponent`.